tensorflow/python/keras/callbacks.py

```diff
@@ -1308,12 +1308,12 @@ def on_epoch_end(self, epoch, logs=None):
 @keras_export('keras.callbacks.LearningRateScheduler')
 class LearningRateScheduler(Callback):
   """Learning rate scheduler.
-
-  Arguments:
-      schedule: a function that takes an epoch index as input
-          (integer, indexed from 0) and returns a new
-          learning rate as output (float).
-      verbose: int. 0: quiet, 1: update messages.
+  
+  A callback for scheduling to update the optimizer's learning rate at the start of each epoch. 
+  The learning rate used in this callback replaces the default learning rate set by the gradient descent optimization algorithm.
```

Comment:
I think the optimization is not necessarily gradient descent. Rephrase it to avoid confusion?